README.md

```diff
@@ -72,6 +73,7 @@ fastify.register(require('fastify-rate-limit'), {
 
 - `global` : indicates if the plugin should apply the rate limit setting to all routes within the encapsulation scope
 - `max`: is the maximum number of requests a single client can perform inside a timeWindow. It can be a sync function with the signature `(req, key) => {}` where `req` is the Fastify request object and `key` is the value generated by the `keyGenerator`. The function **must** return a number.
+- `ban`: is the maximum number of 429 responses to return to a single client before returning 403. When the ban limit is exceeded the context field will have `ban=true` in the errorResponseBuilder
```

Comment:
We should add the info for "distributed application" like:

```
by default this plugins uses an in-memory store, which is fast but if you application works on more than one server it is useless, since the data is store locally.
```

Reply:
Added the extra note at the top of the Options section to provide some detail about that.